packages/rest/src/types.ts

```diff
@@ -107,6 +107,10 @@ export interface RequestBodyValidationOptions extends ajv.Options {
    * - `string[]`: Add an array of keywords from `ajv-keywords`
    */
   ajvKeywords?: true | string[];
+  /**
+   * Enable error message customizing.
+   */
+  ajvTranslate?: (errors: ajv.ErrorObject[]) => ajv.ErrorObject[];
```

Comment:
Should we allow asynchronous translation here?

Reply:
@raymondfeng Yes, it would be more unified, but I think I have to touch a couple more files to do that.

Is that OK?

Reply:
So far, we support synchronous validation only, therefore `validateValueAgainstSchema` is sync too. I feel introducing async translation would require too many changes, I am not convinced the (speculative) benefits are worth the cost.

Let's start with sync translations for the first version, we can look into async translations later, if there is need.

Reply:
I was worried about the probably low ROI of this async change, but I don't like to support both sync & async transformation in the future either.

But I think minimizing changes is better for now.